tests/assembly/targets/targets-elf.rs

```diff
@@ -441,6 +441,15 @@
 //@ revisions: riscv64imac_unknown_none_elf
 //@ [riscv64imac_unknown_none_elf] compile-flags: --target riscv64imac-unknown-none-elf
 //@ [riscv64imac_unknown_none_elf] needs-llvm-components: riscv
+//@ revisions: riscv32e_unknown_none_elf
```

Comment:
It might be a good idea to move these new comments above so that the list is ordered.